packages/playback-core/src/index.ts

```diff
@@ -318,6 +333,21 @@ export const loadMedia = (
 
     hls.loadSource(src);
     hls.attachMedia(mediaEl);
+    mediaEl.addEventListener("canplay", () => {
+      mediaEl.querySelectorAll("track").forEach((track) => {
+        if (track.getAttribute("label") === "thumbnails") {
+          const thumbnailsSrc = track.getAttribute("src");
+          if (!thumbnailsSrc) return;
+          track.removeAttribute("src");
+          // NOTE: This is super hacky. Need to hunt down if there's a
+          // better event to ensure this happens late enough to not
+          // be stomped on by hls.js (CJP)
+          setTimeout(() => {
+            track.setAttribute("src", thumbnailsSrc);
+          }, 500);
```

Comment:
question: Why is this necessary?

Reply:
hls really wants to manage exactly one "selected track" and does silly things asynchronously when you provide "out of band" tracks via `<track/>`. It's also especially not great for metadata text tracks (or chapters etc.) unless it "knows about them" and built them itself. tldr - we end up with no `activeCues` because of disable cases and simply re-setting `src` after hls is done with its initial crud (see event deets in comment) was the most reliable way of getting them loaded (moreso than mucking around with the `mode`)